Core/Object Arts/Dolphin/System/Win32/HttpServer/WinHttpServer.HTTP_SERVICE_CONFIG_CACHE_SET.cls

```diff
@@ -13,13 +13,13 @@ WinHttpServer.HTTP_SERVICE_CONFIG_CACHE_SET guid: (Core.GUID fromString: '{7d7f2
 WinHttpServer.HTTP_SERVICE_CONFIG_CACHE_SET comment: '`HTTP_SERVICE_CONFIG_CACHE_SET` is an `External.Structure` class to wrap the struct ''WinHttpServer.HTTP_SERVICE_CONFIG_CACHE_SET'' from type information in the ''Windows Http Server API'' library.
```

Comment:
Any class comment that says "This comment was automatically generated from a type library" in it, should not be changed. There is no point, as it will be overwritten if regenerated.